packages/@react-spectrum/progress/docs/ProgressBar.mdx

```diff
@@ -0,0 +1,125 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {ProgressBar} from '@react-spectrum/progress';
+```
+
+# ProgressBar
+
+<p>{docs.exports.ProgressBar.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressBar']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/bar-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressBar label="Loading…" value={50} />
+```
+
+## Value
+
+Progress Bars are controlled with the `value` prop representing current percentage (as an integer).
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<ProgressBar label="Loading…" value={25} />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressBar label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+Values are formatted by default as a percentage, but can modified via `formatOptions` for different formats.
+The `formatOptions` are displayed via the [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) applied for the current locale.
+
+```tsx example
+<ProgressBar label="Loading…" formatOptions={{style: 'currency', currency: 'JPY'}} value={60} />
+```
+
+### Accessibility
+
+Depending on the visualization being used (i.e. depending on the `showValueLabel` prop), a `label`, `aria-label`, or `aria-labelledby` attribute are required.
+
+```tsx example
+<div>
+  <ProgressBar aria-labelledby="pc-label" showValueLabel={false} value={45} />
+  <label id="pc-label" style={{display: 'block'}}>Loading…</label>
+</div>
+```
+
+If using the over background variant, ensure the background offers enough contrast for the Progress Bar to be legible and meets color contrast guidelines.
+
+### Internationalization
+
+To internationalize a Progress Bar, a localized string should be passed to the `label` prop, `aria-label` prop, or value of associated `aria-labelledby` element.
```

Comment:
Same as above, mind adding "value of **the** associated" applied in https://github.com/adobe-private/react-spectrum-v3/pull/243 here as well